Source/WebCore/platform/skia/SkiaAllocatorFastMalloc.cpp

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (C) 2024 Igalia S.L.
+ *
+ *  This library is free software; you can redistribute it and/or
+ *  modify it under the terms of the GNU Lesser General Public
```

Comment:
Why is this LGPL?

Reply:
We use the LGPL in the API layer of the GTK and WPE ports and I have my editor automatically insert that as a template, which I forgot to change for this new source file.

Indeed, this is cross-port code so it should use the BSD license; I will change it.